tools/shared/src/main/scala/org/scalajs/core/tools/classpath/IRClasspath.scala

```diff
@@ -42,8 +43,8 @@ final class IRClasspath(
     if (scalaJSIR.nonEmpty) {
       val semantics = Semantics.compliantTo(requiredCompliance.map(_.semantics))
       val output = WritableMemVirtualJSFile(outName)
-      new ScalaJSOptimizer(semantics).optimizeCP(this,
-          Config(output), NullLogger)
+      new ScalaJSOptimizer(semantics, OutputMode.ECMAScript51Global).optimizeCP(
+          this, Config(output), NullLogger)
```

Comment:
Not to break third party code using this and somehow relying on the legacy output. But maybe that's being a bit too safe ^^ I'll change it.